### PR TITLE
ENDOC-580 Add notes re ms/mfe stack options, refine component gen ove…

### PR DESCRIPTION
**Pull request by nshaw**
…rview, fix syncNext cmd

### CODE REVIEW

File: vuepress/docs/next/docs/create/component-gen-overview.md [hunk 1]
```diff
@@ -23,22 +23,20 @@ The component generator has two main objectives
...
    - The code is meant to be edited and enhanced by developers. JHipster provides interactive updates and merges if you need to re-run the generator or add new functionality via JHipster
    - The microservice includes liquibase and integrated data mocking if you want to generate data in your API calls
 
 2. Define bundle and deploy assets
-   - The Entando blueprint includes scripts and tools to automatically generate a bundle ready for deployment to the ECR. The bundle will include your microservice, generated microfrontends, and any microfrontends that you have created manually under the `ui` folder
-   - The generated Spring Boot microservice can be built and deployed to a Docker registry using the provided maven Jib plugin
-   - Generated bundles can be pushed to a Git repository of the user's choice
+   - The Entando Blueprint generates code compatible with the docker-based bundle project structure introduced in Entando 7.1. The bundle project will include a microservice, entity-level microfrontends, as well as other components you have added manually to the project.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/docs/create/component-gen-overview.md:33]: ...a microservice and entity-level micro frontends as well as other components that have been added manually

File: vuepress/docs/next/docs/create/component-gen-overview.md [hunk 2]
```diff
@@ -23,22 +23,20 @@ The component generator has two main objectives
...
+   - The generated Spring Boot microservice can be built and deployed to a Docker registry using the provided build structure
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/docs/create/component-gen-overview.md:34]: this reads like the build structure performs build and deploy... maybe rephrase to clarify how the build structure enables build/deploy

File: vuepress/docs/next/docs/create/component-gen-overview.md [hunk 3]
```diff
@@ -23,22 +23,20 @@ The component generator has two main objectives
...
+   - The bundle itself is pushed to the same Docker registry
 
 3. Deploy bundles to the [Local Hub](../compose/local-hub-overview.md)
-   - Once your Docker images are pushed to your registry and your bundle is available in Git you can generate an Entando bundle descriptor and push the bundles to the ECR
-   - The bundles can be versioned and updated using tags in the Git repository
+   - Once your Docker images are pushed to your registry you can deploy the bundle to the Local Hub in an Entando Application
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/docs/create/component-gen-overview.md:38]: minor, but take the emphasis off of "you" to make instructions generalized and direct: "Once the Docker images are pushed to a registry, the bundle can be deploy to the Local Hub of an Entando Application"

File: vuepress/docs/v7.1/docs/create/component-gen-overview.md [hunk 4]
```diff
@@ -23,22 +23,20 @@ The component generator has two main objectives
...
    - The code is meant to be edited and enhanced by developers. JHipster provides interactive updates and merges if you need to re-run the generator or add new functionality via JHipster
    - The microservice includes liquibase and integrated data mocking if you want to generate data in your API calls
 
 2. Define bundle and deploy assets
-   - The Entando blueprint includes scripts and tools to automatically generate a bundle ready for deployment to the ECR. The bundle will include your microservice, generated microfrontends, and any microfrontends that you have created manually under the `ui` folder
-   - The generated Spring Boot microservice can be built and deployed to a Docker registry using the provided maven Jib plugin
-   - Generated bundles can be pushed to a Git repository of the user's choice
+   - The Entando Blueprint generates code compatible with the docker-based bundle project structure introduced in Entando 7.1. The bundle project will include a microservice, entity-level microfrontends, as well as other components you have added manually to the project.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/v7.1/docs/create/component-gen-overview.md:33]: above comments apply to lines 33-38

File: vuepress/docs/next/docs/create/component-gen-overview.md (hunk 2 above)
Comment (nshaw) [reply vuepress/docs/next/docs/create/component-gen-overview.md:34]: Or drop the build structure phrase...

**Review by Lyd1aCla1r3 — APPROVED** [review]
